src/Native/build-native.cmd

```diff
@@ -151,7 +151,7 @@ if "%__BuildArch%" == "arm64" (
     set __msbuildArgs=/p:Platform=%__BuildArch% /p:PlatformToolset="%__PlatformToolset%"
 )
 
-call %__rootDir%/run.cmd build-managed -project="%__IntermediatesDir%\install.vcxproj" -- /t:rebuild /p:Configuration=%CMAKE_BUILD_TYPE% %__msbuildArgs%
+call msbuild "%__IntermediatesDir%\install.vcxproj" /t:rebuild /p:Configuration=%CMAKE_BUILD_TYPE% %__msbuildArgs%
```

Comment:
I was about to ask if we were sure that msbuild is in the path but I see the command to set the environment above on this file :)